io.openems.edge.core/bnd.bnd

```diff
@@ -18,9 +19,11 @@ Private-Package: \
 	io.openems.edge.scheduler.api;version=latest,\
 	com.google.gson,\
 	slf4j.api,\
-	io.openems.wrapper.sdnotify;version=latest
+	io.openems.wrapper.sdnotify;version=latest,\
+	io.openems.edge.evcs.api;version=latest
```

Comment:
I dont' think this is required... it might be left over from your test to include EVCS to _sum component.